homeassistant/components/tado/climate.py

```diff
@@ -289,7 +289,12 @@ def preset_modes(self):
 
     def set_preset_mode(self, preset_mode):
         """Set new preset mode."""
-        pass
+        if preset_mode == PRESET_AWAY:
```

Comment:
```suggestion
        self._tado.set_presence(preset_mode)
```

@jasperro  per @springstan  question.  I think you can pass this along since you already check it in `set_presence`